style/modules/testimonial.css

```diff
@@ -0,0 +1,83 @@
+/* Desktop */
+/* Background Box */
+
+.testimonial {
+    margin-top: 100px;
+    max-width: 610px;
+    background: white;
+    border: 0.5px solid black;
+}
+
+/* Testimonial Picture */
+
+.testimonial--pic {
+    position: relative;
+    width: 200px;
+    height: 200px;
+    left: 50%;
+    margin-left: -100px;
+    transform: translateY(-80px);
+    overflow: hidden;
+}
+
+.testimonial--pic > img {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    margin: auto;
+    border-radius: 50%;
+    top: -100%;
+    left: -100%;
+    right: -100%;
+    bottom: -100%;
+}
+
+/* Text Container */
+
+.testimonial--content {
+    margin-top: -60px;
```

Comment:
I'd suggest that we move the text like this:

- use `position: absolute` on `.testimonial--pic` to take the image container out of the "document flow." i.e. it won't take up space, and the text won't be pushed down by the image.
  - note that for this to work properly `.testimonial` will need `position: relative` added (this creates a positioning context for use with children)
- use `padding-top` to move the text down

It might seem like 6 of one / half-dozen of the other, but the advantage to the above is that the text spacing and the image size become independent -- you can change the image size without moving the text.

Also, I'd argue that *conceptually* we're moving that image out of the flow, so it's best to do it explicitly in code. (And in general, with CSS, you take things out of the flow using `absolute`.)